Src/BlazorApp/Services/RaceService.cs

```diff
@@ -89,14 +91,34 @@ public List<Race> GetRaces()
 
     public List<Race> GetOngoingRaces()
     {
-        var races = _ctx.Races.Where(r => r.EndRace == null && r.StartRace.HasValue).ToList();
+        Expression<Func<Race, bool>> onGoingRacesFilter = race => race.StartRace == null && race.EndRace == null;
```

Comment:
@terjetopland  Example of using a named filter with Linq;

Reply:
line 94 should be Expression<..............................race.StartRace != null.....;